website/docs/docs/partnerships.mdx

```diff
@@ -154,15 +146,17 @@ HashiCorp Terraform has an extremely wide community of users and contributors, a
 
 Vendors who choose not to support their provider and prefer to make it a community-supported provider will no longer be listed as Partner.
```

Comment:
```suggestion
Vendors who choose not to support their provider and prefer to make it a community-supported provider cannot be Partners.
```

Suggest changing "will no longer be listed as Partner" to "cannot be Partners" to avoid future tense and passive voice style issues.

Reply:
@mgarrell777 -- they can still be technology partners but their provider will be de-listed as "partner" supported tag.

could you help me phrase this better?